src/frontend/preloads/exportNode.ts

```diff
@@ -42,6 +42,27 @@ function handleHiddenMenuCode(event: KeyboardEvent): any {
   }
 }
 
+function myFocusFunction(event: Event): void {
+  const element = event.target as HTMLElement
+  if (element.tagName === 'INPUT') {
+    const inputElement = element as HTMLInputElement
+    if (inputElement.type === 'text' || inputElement.type === 'password') {
+      fetch('http://localhost:9000/keyboard/open', { method: 'GET', mode: 'no-cors' })
```

Comment:
Could you define a constant for http://localhost:9000/keyboard/
On event better (if you have more time) an class that implement an interface with open, close and toggle